lib/functions/image/partitioning.sh

```diff
@@ -214,17 +214,39 @@ function prepare_partitions() {
 						echo "$bootpart : name=\"bootfs\", start=${next}MiB, size=${BOOTSIZE}MiB, type=${type}"
 						local next=$(($next + $BOOTSIZE))
 					else
-						# no `size` argument mean "as much as possible"
+						# No 'size' argument means "expand as much as possible"
 						echo "$bootpart : name=\"bootfs\", start=${next}MiB, type=${type}"
 					fi
 				fi
 
 				# Root filesystem partition
 				if [[ -n "$rootpart" ]]; then
```

Comment:
if you declare rootpart as a local variable, ~ line 124 the user extensions could redefine it if they wanted to move it off the default.